src/AppBundle/Entity/JemarcheReport.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * JemarcheReport
+ *
+ * @ORM\Table(name="jemarche_report")
+ * @ORM\Entity()
+ */
+class JemarcheReport
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="actionType", type="string", length=50)
+     */
+    private $actionType;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="convincedContact", type="text")
+     */
+    private $convincedContact;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="almostConvincedContact", type="text")
```

Comment:
You need to add asserts for form validation (I suppose you were planning it :) ).